docker/cloudbuild.yaml

```diff
@@ -4,45 +4,82 @@
 steps:
 # Build all supported versions.
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
If we want these to build concurrently, these should each have "waitFor: ['-']", otherwise 1.12.6 won't start until 1.9.1 is done.